cookbook-react/src/recipes/collapses/collapse-cmpc/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ReactNode, useState } from 'react';
+import cn from 'classnames';
+
+import styles from './styles.module.scss';
+
+interface CollapsibleProps {
+  children?: ReactNode;
+  title?: string;
+  className?: string;
+  isCollapsed?: boolean;
+  actions?: ReactNode;
+}
+
+function Collapsible({ title, className, isCollapsed, children, actions }: CollapsibleProps) {
+  const [collapsed, setIsCollapsed] = useState(isCollapsed);
+
+  const handleCollapse = () => {
+    setIsCollapsed(!collapsed);
+  };
+
+  return (
+    <div className={`${className ? className : ''} ${styles.collapsibleContainer} column`}>
```

Comment:
`className={cn(className, styles.collapsibleContainer, 'column')}`

`classnames` handles the case when the a class is undefined